EDITORS.md

```diff
@@ -185,6 +185,28 @@ Note that there might be overlapping functionality when using it with RubyMine,
 
 [Ruby LSP plugin](https://plugins.jetbrains.com/plugin/24413-ruby-lsp)
 
+## Kate
+
+[Kate - LSP Client Plugin](https://docs.kde.org/stable5/en/kate/kate/kate-application-plugin-lspclient.html) runs solargraph server by default.
+To set it up with ruby-lsp you need to put that in your LSPClient plugin's "User Server Settings":
+
+```json
+{
+  "servers":
+  {
+    "ruby": {
+      "command": ["ruby-lsp"],
+      "rootIndicationFileNames": ["Gemfile", "Gemfile.lock", "gems.rb", "gems.lock", "Rakefile"],
```

Comment:
Let's remove `Rakefile` here. I know the current `solargraph` config has it, but the others are sufficient for identifying a Ruby codebase.

Reply:
I changed the proposed user configuration to only include necessary items and removed the `rootIndicationFileNames` entirely. The items of the user configuration are merged with the standard configuration, so that unchanged items can be omitted. This is also reflected in the description text.